src/gluonnlp/data/utils.py

```diff
@@ -399,3 +400,23 @@ def __call__(self, s):
             List of strings. Obtained by calling s.split(separator).
         """
         return s.split(self._separator)
+
+def _convert_to_unicode(text):
```

Comment:
instead of having python version inside the function, consider having it outside, so that the version check only happens once.